contrib/asan/Make.user.tools

```diff
@@ -1,2 +1,6 @@
 USE_BINARYBUILDER_LLVM=1
 BUILD_LLVM_CLANG=1
+
+# Enable Julia assertions and LLVM assertions
+FORCE_ASSERTIONS=1
+LLVM_ASSERTIONS=1
```

Comment:
```suggestion
```
You don't need to install `clang` with julia assertions enabled.

Reply:
Oh, is the issue specifically when trying to build a new clang from source with LLVM assertions?

Reply:
This file does not build llvm from source

Reply:
What is this file used for?